.github/workflows/automatic-update.yml

```diff
@@ -43,3 +44,15 @@ jobs:
 
             **Notes for reviewer:**
             This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files, grafana-plugin-sdk-go and docker-compose files for each plugin example
+
+      - name: Auto approve PR
+        uses: hmarr/auto-approve-action@v4
```

Comment:
Do we need a 3rd party action to do this? You're using the gh cli to auto merge the PR could we do the same with:

```
run: gh pr review ${{ steps.create_pr.outputs.pull-request-url }} --approve -b "Approving update"
```